src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,86 @@
+import { useContext, useState } from "react";
+import { deleteComment } from "../api";
+import { UserContext } from "../contexts/User";
+
+const DeleteComment = ({
+  articleComments,
+  setArticleComments,
+  setDeleteCommentID,
+  deleteCommentID,
+}) => {
+  const userValue = useContext(UserContext);
+  const [disableButton, setDisableButton] = useState(false);
+  const [error, setError] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const targetArticle = function (array) {
+      for (let i = 0; i < array.length; i++) {
+        if (
+          array[i].comment_id === Number(deleteCommentID) &&
+          array[i].author === userValue.user
+        ) {
+          return true;
+        }
+      }
+      return false;
+    };
+
+    if (targetArticle(articleComments)) {
+      deleteComment(deleteCommentID).then(() => {
```

Comment:
You'll also need to handle the potential error coming back from a failed delete request. This should have a .catch with some logic for error handling